src/DynamoPackages/PackageManagerSearchElement.cs

```diff
@@ -40,8 +40,12 @@ public int Votes
         public int Downloads { get { return this.Header.downloads; } }
         public string EngineVersion { get { return Header.versions[Header.versions.Count - 1].engine_version; } }
         public int UsedBy { get { return this.Header.used_by.Count; } }
-        public string LatestVersion { get { return Header.versions[Header.versions.Count - 1].version; } }
+        public string LatestVersion { get { return Header.versions != null ? Header.versions[Header.versions.Count - 1].version : String.Empty; } }
         public string LatestVersionCreated { get { return Header.versions[Header.versions.Count - 1].created; } }
+        /// <summary>
+        /// The current selected package version of the user un the Package Manager UI
+        /// </summary>
+        public string SelectedVersion { get; set; }
```

Comment:
hmmm these are changes from https://github.com/DynamoDS/Dynamo/pull/14355 that leaked here?

Reply:
Yes, absolutely! I will fix the conflicts as we get to it.

Reply:
Got the conflicts fixed now @QilongTang , it took me awhile to reconcile the differences, apologies for that.